chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/DeveloperController.java

```diff
@@ -287,10 +287,12 @@ public PermissionDeletedResponse deleteUserFromDeveloper(
         return DeveloperAttestationSubmissionResults.builder()
                 .developerAttestations(attestationManager.getDeveloperAttestations(developerId))
                 .canSubmitAttestationChangeRequest(attestationManager.canDeveloperSubmitChangeRequest(developerId))
+                .submittablePeriod(attestationManager.getSubmittalPeriod(developerId))
```

Comment:
I am having trouble with the naming of these fields because they are so close. "submittablePeriod" and "submittalPeriod". Are there any other options here? Could "submittablePeriod" just be "period"?